src/main/java/httpServer/http/ResponseBuilder.java

```diff
@@ -0,0 +1,27 @@
+package httpServer.http;
+
+import java.util.List;
+
+public class ResponseBuilder {
+
+
+    public String buildResponse(StatusCodes statusCode, List methods){
+        StringBuilder responseStringBuilder = new StringBuilder();
+        responseStringBuilder.append(statusCode.httpResponse);
+        responseStringBuilder.append(Constants.CRLF);
+        if (!statusCode.equals(StatusCodes.PAGE_NOT_FOUND)) {
+            responseStringBuilder.append("Allow: ");
+            for (int i = 0; i < methods.size(); i++) {
+                responseStringBuilder.append(methods.get(i));
+                if (i < (methods.size() - 1)) {
+                    responseStringBuilder.append(", ");
+                }
+            }
+            responseStringBuilder.append(Constants.CRLF);
+        }
+        responseStringBuilder.append("Content-Length: 0");
+        responseStringBuilder.append(Constants.CRLF);
+        return responseStringBuilder.toString();
```

Comment:
I broke out each line into a separate private function. I'm getting towards the end of my sprint so I'm going to err on the side of renaming rather than trying to fully implement `Builder` pattern if that's okay.